docs/settings/general-infra-logs-ui-settings.asciidoc

```diff
@@ -12,6 +12,6 @@
 
 `xpack.infra.sources.default.fields.host`:: Field used to identify hosts. Defaults to `beat.hostname`.
```

Comment:
This doesn't match the actual default anymore. Do we want to fix that here as well?

Reply:
Good catch... I did another search through the source for `beat.hostname` and everything else is part of the test data. I did the same thing before for `kubernetes.pod.name` and `docker.container.id`.